# HG changeset patch
# User Rik <rik@octave.org>
# Date 1361767113 28800
#      Sun Feb 24 20:38:33 2013 -0800
# Node ID 9720ecf8d25744bca6bba474179f03615420cf68
# Parent  1339c964b5275fcb620e3ad813058f3055211929
Fix additional warning associated with Texinfo 5.0 (bug #38392)

* doc/interpreter/octave.texi: Add 3 nodes missing from plot @menu
* doc/interpreter/plot.txi: Add 3 nodes missing from plot @menu

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -493,16 +493,19 @@ Plotting
 
 High-Level Plotting
 
 * Two-Dimensional Plots::       
 * Three-Dimensional Plots::  
 * Plot Annotations::            
 * Multiple Plots on One Page::  
 * Multiple Plot Windows::       
+* Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions
+* Manipulation of Plot Windows::
+* Use of the @code{interpreter} Property::
 * Printing and Saving Plots::              
 * Interacting with Plots::      
 * Test Plotting Functions::     
 
 Two-Dimensional Plots
 
 * Axis Configuration::  
 * Two-dimensional Function Plotting::  
diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -57,16 +57,19 @@ If you need more detailed control, see @
 and @ref{Advanced Plotting}.
 
 @menu
 * Two-Dimensional Plots::       
 * Three-Dimensional Plots::  
 * Plot Annotations::            
 * Multiple Plots on One Page::  
 * Multiple Plot Windows::       
+* Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions::
+* Manipulation of Plot Windows::
+* Use of the @code{interpreter} Property::
 * Printing and Saving Plots::              
 * Interacting with Plots::      
 * Test Plotting Functions::     
 @end menu
 
 @node Two-Dimensional Plots
 @subsection Two-Dimensional Plots
 
@@ -512,33 +515,34 @@ fplot (@@cos, [-10, 10]);
 @end group
 @end example
 
 @noindent
 creates two figures, with the first displaying a sine wave and
 the second a cosine wave.  Figure numbers must be positive integers.
 
 @DOCSTRING(figure)
-@subsection Use of @code{axis},  @code{line}, and @code{patch} functions
-@cindex use of @code{axis},  @code{line}, and @code{patch} functions
+
+@node Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions
+@subsection Use of @code{axis}@comma{} @code{line}@comma{} and @code{patch} Functions
 
 You can create axes, line, and patch objects directly using the
 @code{axes}, @code{line}, and @code{patch} functions.  These objects
 become children of the current axes object.
 
 @DOCSTRING(axes)
 @DOCSTRING(line)
 @DOCSTRING(patch)
 
 @DOCSTRING(fill)
 
 @DOCSTRING(surface)
 
-@subsection Manipulation of plot windows
-@cindex manipulation of plot windows
+@node Manipulation of Plot Windows
+@subsection Manipulation of Plot Windows
 
 By default, Octave refreshes the plot window when a prompt is printed,
 or when waiting for input.  The
 @code{drawnow} function is used to cause a plot window to be updated.
 
 @DOCSTRING(drawnow)
 
 Only figures that are modified will be updated.  The @code{refresh}
@@ -586,18 +590,18 @@ figure window, call the @code{close} fun
 @DOCSTRING(shg)
 
 @DOCSTRING(delete)
 
 @DOCSTRING(close)
 
 @DOCSTRING(closereq)
 
+@node Use of the @code{interpreter} Property
 @subsection Use of the @code{interpreter} Property
-@cindex use of the @code{interpreter} property
 
 All text objects, including titles, labels, legends, and text, include
 the property 'interpreter', this property determines the manner in which
 special control sequences in the text are rendered.  If the interpreter
 is set to 'none', then no rendering occurs.  At this point the 'latex'
 option is not implemented and so the 'latex' interpreter also does not
 interpret the text.
 
